GetCertificate/certificate.headerInfo.ts

```diff
@@ -0,0 +1,78 @@
+import { PreferredLanguageEnum } from "@pagopa/io-functions-commons/dist/generated/definitions/PreferredLanguage";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { pipe } from "fp-ts/lib/function";
+
+import * as O from "fp-ts/lib/Option";
+import { match } from "ts-pattern";
+import { HeaderInfo } from "../generated/definitions/HeaderInfo";
+
+import {
+  DefaultLanguage,
+  isSupportedLanguage,
+  SupportedLanguage
+} from "../utils/conversions";
+
+import {
+  Certificates,
+  RecoveryCertificate,
+  TestCertificate,
+  VacCertificate
+} from "./certificate";
+
+const EUROPEAN_LOGO_ID = "1" as NonEmptyString;
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const ITALIAN_LOGO_ID = "2" as NonEmptyString;
+
+/* 
+Title and subtitle of certificate header 
+for each supported languageF
+*/
+
+const standardTitleAndSubtitle: {
+  [key in SupportedLanguage]: Omit<HeaderInfo, "logo_id">;
+} = {
+  [PreferredLanguageEnum.it_IT]: {
+    subtitle: "",
+    title: "Certificazione verde COVID-19" as NonEmptyString
+  },
+  [PreferredLanguageEnum.en_GB]: {
+    subtitle: "",
+    title: "EU Digital COVID Certificate" as NonEmptyString
+  },
+  [PreferredLanguageEnum.de_DE]: {
+    subtitle: "",
+    title: "Grünes COVID-19-Zertifikat" as NonEmptyString
+  }
+};
+
+const getStandardHeader = (language: SupportedLanguage): HeaderInfo => ({
+  ...standardTitleAndSubtitle[language],
+  logo_id: EUROPEAN_LOGO_ID
+});
+
+const emptyHeader: HeaderInfo = {
+  logo_id: "",
+  subtitle: "",
+  title: ""
+};
+
+export const getHeaderInfoForLanguage = (
+  lang: O.Option<PreferredLanguageEnum>
+) => (certificate: O.Option<Certificates>): HeaderInfo => {
```

Comment:
Code design: 
we can make this function much simpler if expect a `Certificates` instead of both `Some<Certificates>` and `None`; we can export `emptyHeader` and let the handler decide.
Both this module and handler would be more straightforward to read, hiding less each others ;)

(more in general, if we find ourselves in the need of passing a complex type to a function - Option, Either, etc - it's almost always the case we are dealing with a function that has more than one responsibility)

Reply:
You're right, thanks. Done in [72615ac](https://github.com/pagopa/io-functions-eucovidcerts/pull/83/commits/72615ac3d745df4d608bd36697c400c9f0ce1c2f)